examples/src/operation/bootstrap/bootstrapPool.ts

```diff
@@ -0,0 +1,64 @@
+import { Bootstrap, poolUtils } from "@tinymanorg/tinyman-js-sdk";
+import { getAccount } from "../../util/account";
+import { getAssets } from "../../util/asset";
+import { algodClient } from "../../util/client";
+import signerWithSecretKey from "../../util/initiatorSigner";
+import { SDK_TEST_ARGS } from "../../util/other";
+
+/**
+ * Creates (i.e. "Bootstraps") a pool with an owned asset pair
+ */
+export async function bootstrapPool() {
+  const account = await getAccount();
+  const initiatorAddr = account.addr;
+  const { ids: assetIds } = await getAssets();
+  const [asset1ID, asset2ID] = assetIds;
+  const assetA = await algodClient.getAssetByID(asset1ID).do();
+  const assetB = await algodClient.getAssetByID(asset2ID).do();
+  const [asset_1, asset_2] = [assetA, assetB].map((asset) => ({
+    id: String(asset.index),
+    unit_name: asset.params["unit-name"],
+  }));
+
+  const poolInfo = await poolUtils.v2.getPoolInfo({
+    ...SDK_TEST_ARGS,
+    asset1ID,
+    asset2ID,
+  });
+
+  if (!poolUtils.isPoolNotCreated(poolInfo)) {
+    console.log("⚠️ Pool already exists, skipping bootstrap.");
+    return;
+  }
+
+  const bootstrapTxns = await Bootstrap.v2.generateTxns({
+    ...SDK_TEST_ARGS,
```

Comment:
I think we should just repeat these to make it easier to follow.
```suggestion
      network: "testnet" as SupportedNetwork,
      client: algodClient,
```

and actually since SDK_TEST_ARGS involves the contractVersion, it is passing a redundant argument